x/wasm/internal/keeper/genesis.go

```diff
@@ -48,11 +56,15 @@ func ExportGenesis(ctx sdk.Context, keeper Keeper) types.GenesisState {
 
 	maxCodeID := keeper.GetNextCodeID(ctx)
 	for i := uint64(1); i < maxCodeID; i++ {
+		if !keeper.containsCodeInfo(ctx, i) {
```

Comment:
okay, see how this can be slow if there are *huge* gaps. But this is export and only if someone decides to play around at genesis time